libraries/abstractions/common_io/test/iot_assisted_tests/adc/test_iot_adc_rp3.py

```diff
@@ -0,0 +1,57 @@
+# Amazon FreeRTOS V1.2.3
```

Comment:
In order to make these scripts easier to find, I believe we should place these scripts one level up, in a folder called <common-io>/test_scripts. @qiutongs @vjjunnut @cobusve @cmcgcc thoughts?

Reply:
Agreed.

Reply:
These scripts are still related to tests. So I would still be inclined to keep them under common-io/test/. But may be rename them to test_scipts instead of iot_assisted_tests

Reply:
ok, if required, it is easy to change later.